playbooks/common/openshift-cluster/upgrades/v3_10/upgrade_control_plane.yml

```diff
@@ -115,6 +116,15 @@
   - role: openshift_sdn
     when: openshift_use_openshift_sdn | default(True) | bool
 
+- name: Run kube proxy, needed for ACI
+  hosts: oo_masters
```

Comment:
Does it need to run on all masters? Afaik it sufficient to run it on first master only

Reply:
Yes, you are right, will change it to "oo_first_master"